src/taipy/core/_repository/_fs_base.py

```diff
@@ -49,7 +50,7 @@ def object_hook(self, source):
             return source
 
 
-class _FileSystemRepository(Generic[ModelType, Entity]):
+class _FileSystemRepository(_AbstractRepository[ModelType, Entity]):
```

Comment:
I would rename the file to `_file_system_repository`. Is there any reason to keep `fs_base` name?

Reply:
Agreed.